content/hi/docs/reference/glossary/workload.md

```diff
@@ -0,0 +1,23 @@
+---
+title: कार्यभार(Workload)
+id: कार्यभार(Workloads)
```

Comment:
This `id` is wrong. The `id` should be like this (same as the English original):
```suggestion
id: workloads
```